psi4/driver/schema_wrapper.py

```diff
@@ -167,13 +169,22 @@ def _convert_variables(data, context=None, json=False):
     ret = {}
     for key, var in needed_vars.items():
 
+        conversion_factor = var.get("conversion_factor", 1.0)
+
         # Get the actual variables
         if isinstance(var["variables"], str):
             value = data.get(var["variables"], None)
+            if value:
+                if isinstance(value, (int, float)):
+                    value *= conversion_factor
+                elif isinstance(value, (core.Matrix, core.Vector)):
+                    value.scale(conversion_factor)
```

Comment:
always multiply rather than `conversion_factor` devault to None, then `if conversion_factor: convert`?

Reply:
I thought it would be a lot more logic, but isn't too bad.